packages/playground/src/components/app-root/app-root.tsx

```diff
@@ -201,7 +201,7 @@ export class AppRoot {
     const { user } = this.accountState;
     const { signer } = this.walletState;
 
-    const signature = await signer.signMessage(
+    const signature = await signer!.signMessage(
```

Comment:
@snario Let's avoid using `!`. We can do above:

```ts
const signer = this.walletState.signer as Signer;
```

Reply:
Why is this better?